escalation/store.go

```diff
@@ -45,6 +46,7 @@ type PolicyStore interface {
 	FindAllPoliciesBySchedule(ctx context.Context, scheduleID string) ([]Policy, error)
 	FindManyPolicies(ctx context.Context, ids []string) ([]Policy, error)
 	DeleteManyPoliciesTx(ctx context.Context, tx *sql.Tx, ids []string) error
+	FindAllNotices(ctx context.Context, policyID string) ([]notice.Notice, error)
```

Comment:
I feel like maybe all the "notice" type things should live in their own store -- EP store can stick to find/update on the policy itself, but then we can have one place for lookup logic like this.

One way that may be important is that notice lookup logic may get somewhat expensive in the future, and we might be caching them for short time periods -- it would get hard to maintain to spread that across all the different stores rather than keeping those concerns inside a single package for finding "notices".